extensions/wikia/UserLogin/WikiaConfirmEmailSpecialController.class.php

```diff
@@ -78,7 +78,7 @@ public function index() {
 			}
 
 			$expUser = User::newFromConfirmationCode( $this->code );
```

Comment:
Hmm, my opinion is not now, just to keep this PR simple.  Plus I don't really see this as improper verification as much as a more clear verification.